src/Test.FunctionalTests.BinSkim.Rules/RuleTests.cs

```diff
@@ -1166,10 +1214,19 @@ public void BA2025_EnableShadowStack_Fail()
         [Fact]
         public void BA2025_EnableShadowStack_NotApplicable()
         {
-            this.VerifyApplicability(
+            HashSet<string> notApplicableArm64 = new HashSet<string>() { MetadataConditions.ImageIsArm64BitBinary };
+
+            this.VerifyApplicabililtyByConditionsOnly(
```

Comment:
>VerifyApplicabililtyByConditionsOnly [](http://example.com/codeflow?start=17&length=36)

You are getting solidly into our test infrastructure if you understand this pattern. nice work. :) I'm sure Shaopeng may have been helpful in gaining 'the knowledge'.